src/styles/stars.css

```diff
@@ -0,0 +1,37 @@
+/* stars section */
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url(../images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
+
+.stars--1 > .stars__star:first-child,
+.stars--2 > .stars__star:nth-child(-n + 1),
+.stars--2 > .stars__star:nth-child(-n + 2),
+.stars--3 > .stars__star:nth-child(-n + 1),
+.stars--3 > .stars__star:nth-child(-n + 2),
+.stars--3 > .stars__star:nth-child(-n + 3),
+.stars--4 > .stars__star:nth-child(-n + 1),
+.stars--4 > .stars__star:nth-child(-n + 2),
+.stars--4 > .stars__star:nth-child(-n + 3),
+.stars--4 > .stars__star:nth-child(-n + 4),
+.stars--5 > .stars__star:nth-child(-n + 1),
+.stars--5 > .stars__star:nth-child(-n + 2),
+.stars--5 > .stars__star:nth-child(-n + 3),
+.stars--5 > .stars__star:nth-child(-n + 4),
+.stars--5 > .stars__star:nth-child(-n + 5) {
+  background-image: url(../images/star-active.svg);
+}
```

Comment:
You don't have to write a selector for each star
`.stars--4 > .stars__star:nth-child(-n + 4)` means it is a selector for first four elements with such class
`.stars--3 > .stars__star:nth-child(-n + 3),`  means it is a selector for first three elements with such class

so you just need to write one selector per each row of stars and in the end you will receive only 5 strings of selectors